tests/utils.js

```diff
@@ -798,6 +799,17 @@ module.exports = {
       });
   },
 
+  resetBrowserNative: async () => {
+    return await browser.driver
```

Comment:
You should be able to `await` the `utils.resetBrowser` function in your `beforeEach` above.     If you find that's not possible this does not need the return and the await.

Reply:
Actually, nevermind I see why this is necessary.  I do think there is a bit of confusion on the await and returning.   Since no value is being returned and we don't need to block here. I think you can just return the promise.

Reply:
This is no longer needed. You can safely use the existent function.